### ISSUE TITLE
ERROR: Hook 'bootsplash-logo-white' cannot be found

### ISSUE THREAD

**Issue by HuntedNarrator** [holoitsme/manjaro-bootsplash-maker#1]
Hello, and thank you for making this tool! I've followed the instructions to a tee but ran into this error when running `sudo mkinitpcio -P`. My hooks line in mkinitcpio.conf is `HOOKS='quiet base udev autodetect modconf block keyboard keymap filesystems bootsplash-logo-white'`. Any help would be greatly appreciated.

**Comment by federico-ntr** [comment]
Unfortunately, due to work reasons, I'll be stuck with Windows until I get a new computer, so I'm not able to test and resolve issues. 
Maybe I'll get a new computer soon, if you could post here the files you used and the command you ran I'll try as soon as I can. Sorry for trouble.

**Comment by authoranderton** [comment]
Hello again. Backtracking through the process, I think I've found the issue. I used `makepkg -si` out of muscle memory but the README file says to install using pacman -U $package_name. I tried running the entire process again with a new name for the theme, but when I reach the pacman step, it doesn't find the package. Your instructions produced by the readme file are a little vague/unclear. Firtstly, the terminal doesn't like the $ before the package name. Secondly, your README file doesn't explain what the package name should be. I looked through the processes that ran during `makepkg` and tried the two names that pop up there, but it looks like the make process is adding a space in the name that pacman doesn't like, for example `bootsplash-manjaro-logo-white 0.1-1` with a space between the 'e' and the '0'. I tried without the numbers, didn't find package. I tried with an underscore, didn't find. I tried with no space, didn't find. Can you tell me what directory the make process builds the package into so I can find the package and grab the name from there?

**Comment by HuntedNarrator** [comment]
I realized I wasn't capitalizing the E in Envy, so I reran the pacman commands with the capital E, same result:
```
[arthur@ManjaroEnvy ~]$ cd manjaro-bootsplash-maker
[arthur@ManjaroEnvy manjaro-bootsplash-maker]$ cd manjaro-bootsplash-Envy
[arthur@ManjaroEnvy manjaro-bootsplash-Envy]$ sudo pacman -U manjaro-bootsplash-Envy
[sudo] password for arthur: 
loading packages...
error: 'manjaro-bootsplash-Envy': could not find or read package
[arthur@ManjaroEnvy manjaro-bootsplash-Envy]$ sudo pacman -U bootsplash-themes-manjaro-Envy
loading packages...
error: 'bootsplash-themes-manjaro-Envy': could not find or read package
[arthur@ManjaroEnvy manjaro-bootsplash-Envy]$ sudo pacman -U bootsplash-theme-manjaro-Envy
loading packages...
error: 'bootsplash-theme-manjaro-Envy': could not find or read package
[arthur@ManjaroEnvy manjaro-bootsplash-Envy]$ ^C
[arthur@ManjaroEnvy manjaro-bootsplash-Envy]$ 

```
I've also included a screenshot of the README generated by bs_maker.sh
![image](https://user-images.githubusercontent.com/72247250/94953813-01b5cb80-049d-11eb-860e-200d657f4a21.png)

**Comment by HuntedNarrator** [comment]
Found the solution! Paying close attention to the makepkg readouts, your README file gives the wrong name for the HOOK entry. Above it says bootsplash-manjaro-envy because I had initially named the file "manjaro envy". However, renaming it to just "envy", the README says the HOOK should be "bootsplash-envy". This is wrong, as during the `makepkg -si` readouts it says:
` -> Running build hook: [bootsplash-manjaro-envy]
==> Generating module dependencies
==> Creating gzip-compressed initcpio image: /boot/initramfs-5.8-x86_64-fallback.img
==> Image generation successful
`
It clearly shows "manjaro" being added to the hook name, despite your README only saying bootsplash-envy. I'm a noob and it literally took me all day to figure this out. Also, pacman doesn't like to install using `sudo pacman -U` like your readme says. I had to use `makepkg -si` to get it to work. Once I overcame those two roadblocks, all is working great, though. Frustrating as these small errors may be, it's still a great tool and invaluable to make our own custom splashes. Thanks for the hard work on your part!
